src/user_recipe.jl

```diff
@@ -229,8 +229,11 @@ end
     SliceIt, nothing, Volume(vol, args...), nothing
 end
 
-# Dicts: each entry is a data point (x,y)=(key,value)
-@recipe f(d::AbstractDict) = collect(keys(d)), collect(values(d))
+# Dicts: each entry is a data point (x,y)=(key,value), sorted
+@recipe function f(d::AbstractDict)
+    seriestype := :line
```

Comment:
```suggestion
    seriestype --> :line
```
Otherwise you can't change to scatter/bar/... when plotting a dict.

Reply:
I still think we should go for `-->` here.